invenio_records_resources/services/records/results.py

```diff
@@ -119,24 +116,16 @@ def hits(self):
         """Iterator over the hits."""
         for hit in self._results:
             # Load dump
-            record = self._service.record_cls.loads(
-                hit.to_dict()
-            )
+            record = self._service.record_cls.loads(hit.to_dict())
 
             # Project the record
-            links = LinksStore(host=_current_host)
```

Comment:
Minor: Don't we need links here as well? The links facotry injected in the search schema is not used here as far as I can see?

Reply:
Right! I created a separate issue to fix this. It shouldn't block the rest of the functionality.